draft-ietf-scone-protocol.md

```diff
@@ -279,12 +279,17 @@ The most significant bit (0x80) of the packet indicates that this is a QUIC long
 header packet.  The next bit (0x40) is reserved and can be set according to
 {{!QUIC-BIT=RFC9287}}.
 
-The low 6 bits (0x3f) of the first byte contain the Rate Signal field. Values
-for this field are described in {{rate-signal}}.
+The Rate Signal field uses 7 bits spanning the low 6 bits (0x3f) of the first
+byte and the most significant bit of the version field. This creates a
+contiguous 7-bit field with values from 0 to 127. Values for this field are
+described in {{rate-signal}}.
```

Comment:
Can we tie that to the proposed definition of the field as _Rate Signal High Bits (6)_ ?